cluster/gce/util.sh

```diff
@@ -660,7 +660,7 @@ function create-network() {
       --project "${PROJECT}" \
       --network "${NETWORK}" \
       --source-ranges "10.0.0.0/8" \
-      --allow "tcp:1-65535,udp:1-65535,icmp" &
+      --allow "tcp:1-2379,tcp:2382-65535,udp:1-65535,icmp" &
```

Comment:
I guess you'd have to have two separate rules:
* for master node something which explicitly lists ports that we want to expose
* for minion nodes something that will open all ports

Would it work?